semgrep-core/src/fixing/Autofix.mli

```diff
@@ -0,0 +1,6 @@
+val render_fix :
+  Lang.t ->
+  Metavariable.bindings ->
+  fix_pattern:string ->
+  target_contents:string Lazy.t ->
```

Comment:
maybe explain why the lazy content?
The idea that many things can fail before even trying to grab part of the target file (for the metavar), that
it's better to have the content lazy.